dist/App.js

```diff
@@ -0,0 +1,16 @@
+import React, { PureComponent } from 'react';
+import { createAppContainer } from 'react-navigation';
+import { createStackNavigator } from 'react-navigation-stack';
+import { navigationStructure } from './app/modules';
+// import { AppWrapper } from './styled';
+import { View } from 'react-native';
+// import ThemeProvider from 'styled-components';
+// import { defaultTheme } from './app/styles/themes/defaultTheme';
+const AppNavigator = createStackNavigator(navigationStructure);
+const AppContainer = createAppContainer(AppNavigator);
+export default class App extends PureComponent {
```

Comment:
maybe it should be just functional component, because it is doning nothing. else than render